static/bidder-info/appStockSSP.yaml

```diff
@@ -0,0 +1,22 @@
+aliasOf: "teqblaze"
+endpoint: "https://#{REGION}#.al-ad.com/pserver"
```

Comment:
This doesn't look like a valid endpoint. If the region must be configured by the host, please mark this adapter as disabled by default. The linked bidder docs should also provide valid REGION values and/or add a comment in this YAML file with the valid choices.